crates/dapf/src/acceptance/mod.rs

```diff
@@ -166,6 +166,19 @@ pub enum HttpClient {
 
 impl HttpClient {
     fn reused(leader_tls_identity: Option<Identity>) -> Self {
```

Comment:
SSLKEYLOGFILE should not be enabled by default, as it is not secure. The user should have to set it with a flag.

Reply:
even if it's only used from a dapf? I guess we run dapf on the edge. I'll add that flag